test/cpp/qps/client_async.cc

```diff
@@ -54,6 +54,10 @@
 #include "test/cpp/qps/usage_timer.h"
 #include "test/cpp/util/create_test_channel.h"
 
+extern "C" {
```

Comment:
Can you move this extern "C" protection into the included header file? It generally makes more sense there (particularly if the underlying implementation ends up changing to C++)